pkg/operator/workloads/api_workload.go

```diff
@@ -309,6 +315,11 @@ func tfAPISpec(
 						Resources: kcore.ResourceRequirements{
 							Requests: transformResourceList,
 						},
+						Ports: []kcore.ContainerPort{
```

Comment:
Do we need these (and lines 352-356) in `onnxAPISpec()`

Reply:
yes